boxoffice/views/admin_order.py

```diff
@@ -125,23 +125,18 @@ def admin_org_order(org, order):
     return dict(org=org, order=order, line_items=line_items)
 
 
-# This enpoint has been added to fetch details of an order to generate invoice outside Boxoffice.
-# Not to be used within the app.
-@app.route('/api/1/organization/<org_name>/order/<int:receipt_no>', methods=['GET'])
-@load_models(
-    (Organization, {'name': 'org_name'}, 'org'),
-    (Order, {'organization': 'org', 'receipt_no': 'receipt_no'}, 'order')
-)
-def order_api(org, order):
-    check_api_access(org.details.get('access_token'))
-
+def get_order_details(order):
     line_items_list = [{
         'title': li.item.title,
+        'category': li.item.category.title,
+        'event_date': li.item.event_date if li.item.event_date else '',
```

Comment:
If it's a `date` or `datetime` type, use `event_date.isoformat()`. The `else` condition should return `None`, not `''`.